src/app/page.tsx

```diff
@@ -47,21 +54,6 @@ export default function Home() {
           Preorder the Rust Book Now!
         </a>
       </div>
-      <div id='books' className='min-h-dvh'>
```

Comment:
This landing page obscures the full-page versions. Having a more simple landing page will load more quickly, and be more consistent with user expectations around page urls.